tests/unit/worker/job_poller_test.py

```diff
@@ -105,3 +101,22 @@ def test_calculate_max_jobs_to_activate(self, job_poller: JobPoller, active_jobs
         max_jobs_to_activate = job_poller.calculate_max_jobs_to_activate()
 
         assert max_jobs_to_activate == expected
+
+
+@pytest.mark.asyncio
+class TestPoll:
```

Comment:
```suggestion
class TestHandleMaxJobs:
```

Reply:
Thanks, I have changed method name to activate_max_jobs and test class to TestActivateMaxJobs